src/index.html

```diff
@@ -1,18 +1,140 @@
-  <body>
-    <h1>Product cards</h1>
+  <body class="page__body">
+    <main class="main">
+      <div
+        class="product"
+        data-qa="card"
+      >
+        <div class="product__image">
+          <a href="#">
+            <img
+              class="product__photo"
+              src="images/imac.jpeg"
+              alt="iMac"
+            />
+          </a>
+        </div>
+
+        <div class="product__description">
+          <a
+            class="product__title"
+            href="#"
+          >
+            <h3 class="product__good">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h3>
+          </a>
+        </div>
+
+        <div class="product__code">
+          <div class="product__text-code">
+            <p class="product__text-code-paragraph">Product code: 195434</p>
+          </div>
+          <div class="product__number-code"></div>
+        </div>
+
+        <div class="product__stars">
+          <div class="product__picture-stars">
+            <!-- <div class="stars stars--0">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div> -->
+
+            <!-- <div class="stars stars--1">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <div class="stars stars--2">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div> -->
+
+            <!-- <div class="stars stars--3">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div> -->
+
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <!-- <div class="stars stars--5">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div> -->
+          </div>
+          <div class="product__text-stars">
+            <p class="product__text-stars-paragraph">Reviews: 5</p>
+          </div>
```

Comment:
why do you need to wrap with `div`?
```suggestion
            <p class="product__text-stars-paragraph">Reviews: 5</p>
```